config/routes.rb

```diff
@@ -6,7 +6,7 @@
   devise_for :users, skip: [:registrations], controllers: { sessions: "sessions" }
 
   resources :users, only: [:edit, :update]
-  resources :profiles, except: [:show]
+  resources :profiles
```

Comment:
A user has only one profile, i.e. there is no requirement for a list of profiles for a single user. Admin users, however, do change things. The choice here would be splitting that into an admin/profiles resource, which I understand we might not want to do at this point.